index.rst

```diff
@@ -146,6 +146,7 @@ Welcome to the documentation for Citus 7.0! Citus horizontally scales PostgreSQL
 
    reference/citus_sql_reference.rst
    reference/sql_workarounds.rst
+   reference/useful_queries.rst
```

Comment:
This section probably belongs in the Administration section, rather than SQL Reference.